Dockerfile

```diff
@@ -142,10 +139,8 @@ RUN conda install -y python=3.7.1 \
                      graphviz=2.40.1 \
                      traits=4.6.0 \
                      zlib; sync && \
-    chmod -R a+rX /usr/local/miniconda; sync && \
-    chmod +x /usr/local/miniconda/bin/*; sync && \
```

Comment:
Are we sure this glitch has been fixed?

Reply:
our test datasets are passing on Circle - I don't think this is still necessary.

Reply:
yes, but this used to be the case and people would experience linking issues with singularity... not sure it is worth removing.

Reply:
Actually, it would be best to remove in a separate PR, so that it is clear within one atomic change.

Reply:
I think this is the only outstanding issue before merging this PR, correct?